digiroad2-oracle/src/main/resources/db/migration/V236__create_LengthOfRoadAxis_properties.sql

```diff
@@ -0,0 +1,205 @@
+
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'string', 0, 'db_migration_v236', 'additional_information',
+        (select id from LOCALIZED_STRING where VALUE_FI = 'Lisätieto'));
+
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID,
+                      DEFAULT_VALUE)
+VALUES (primary_key_seq.nextval, 450, 'number', 0, 'db_migration_v236', 'state',
+        (select id from LOCALIZED_STRING where VALUE_FI = 'Tila'), 3);
+
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'string', 0, 'db_migration_v236', 'end_date',
+        (select id from LOCALIZED_STRING where VALUE_FI = 'Loppu päivämäärä'));
+
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'string', 0, 'db_migration_v236', 'start_date',
+        (select id from LOCALIZED_STRING where VALUE_FI = 'Alku päivämäärä'));
+
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'string', 0, 'db_migration_v236', 'milled',
+        (select id from LOCALIZED_STRING where VALUE_FI = 'Jyrsitty'));
+INSERT INTO PROPERTY (ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_BY, PUBLIC_ID, NAME_LOCALIZED_STRING_ID)
+VALUES (primary_key_seq.nextval, 450, 'number', 0, 'db_migration_v236', 'condition',
+        (select id from LOCALIZED_STRING where VALUE_FI = 'Kunto'));
+
+
+-- ENUMARATED VALUE CONDITION
+
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 1, 'Erittäin huono', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'condition'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 2, 'Huono', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'condition'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 3, 'Tyydyttävä', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'condition'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 5, 'Hyvä', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'condition'));
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
+VALUES (primary_key_seq.nextval, 6, 'Erittäin hyvä', '', 'db_migration_v236',
+        (SELECT ID FROM PROPERTY WHERE PUBLIC_ID = 'condition'));
+--material
+INSERT INTO ENUMERATED_VALUE (ID, VALUE, NAME_FI, NAME_SV, CREATED_BY, PROPERTY_ID)
```

Comment:
Check the ENUMERATED_VALUE inserted after the corrections above, because for each time we have a single choice type we will have an enumerated to insert.